allure-generator/src/test/javascript/stubs/handlebars-compiler.js

```diff
@@ -5,8 +5,6 @@ const path = require('path');
 const fs = new MemoryFS();
 const file = process.argv[2];
 
-console.log('SETUP HOHOHO');
-
```

Comment:
After this change, I run `yarn test` and it passed localy.
How else should I test?